packages/Breadcrumbs/stories/Breadcrumbs.tests.stories.js

```diff
@@ -19,3 +20,14 @@ screener.story = {
     },
   },
 };
+
+export const rerenderBreadcrumbs = () => <RerenderBreadcrumbs />;
+screener.story = {
+  parameters: {
+    docs: { page: null },
+    options: {
+      isToolshown: true,
+      showPanel: false,
+    },
+  },
```

Comment:
You _could_ `import { testStoryParameters } from "storybook/assets/storyParameters"` and use that here instead.